app/lib/tool/neat_task/pub_dev_tasks.dart

```diff
@@ -20,18 +17,57 @@ import '../../job/backend.dart';
 import '../../package/backend.dart';
 import '../../scorecard/backend.dart';
 import '../../search/backend.dart';
+import '../../service/email/backend.dart';
+import '../../shared/configuration.dart';
 import '../../shared/datastore.dart';
 import '../../shared/integrity.dart';
+import '../../task/backend.dart';
+import '../../task/cloudcompute/googlecloudcompute.dart';
+import '../../task/global_lock.dart';
 import '../../tool/backfill/backfill_new_fields.dart';
 import '../maintenance/remove_orphaned_likes.dart';
 import '../maintenance/update_package_likes.dart';
+import '../maintenance/update_public_bucket.dart';
 
 import 'datastore_status_provider.dart';
 
 final _logger = Logger('pub_dev_tasks');
 
 /// Periodic task that are not tied to a specific service.
 void _setupGenericPeriodicTasks() {
+  // Tries to send pending outgoing emails.
+  _15mins(
+    name: 'send-outgoing-emails',
+    isRuntimeVersioned: false,
+    task: () async {
+      final aquireAbort = Completer();
+      final aquireTimer = Timer(Duration(minutes: 2), () {
```

Comment:
2 minutes? But you keep trying to send for 10 minutes?

I'm confused?

Reply:
I thought that this is a timeout for the acquire operation, and once we have the lock, we use the 20 minutes expiration (but we keep a soft-limit of 10 minutes to send the queue, so will return the lock after about 10 minutes).
Isn't it the case?